src/plugins/timelion/public/directives/_form.scss

```diff
@@ -1,54 +1,36 @@
-// These mixins are temporary helpers to consolidate styles of elements that
-// are not yet converted to use EUI.
-
-// DO NOT CONTINUE TO USE THESE MIXINS
-
-@mixin __legacyInputStyles__bad {
-  &:not([type='range']) {
-    appearance: none;
-  }
+.form-control {
```

Comment:
These styles/file is now in 2 places. Here and `src/legacy/core_plugins/timelion/public/directives/`. Can there just be one?

Reply:
yes, thx, one place of files to rule em all, removed